src/elements/content-sidebar/activity-feed/task-new/TaskDueDate.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../common/variables';
+
+$error-color: $red;
```

Comment:
let's rename this since this sass variable is global and this name is pretty generic. Some SUIT-able name would work

https://sass-lang.com/documentation/variables#scope

Reply:
I took this from the previous file, but do you think there's a reason to rename it? Can we just use `$red` here?